indi-eqmod/skywatcher.h

```diff
@@ -336,6 +336,6 @@ class Skywatcher
 
         bool snapportstatus[NUMBER_OF_SKYWATCHERAXIS];
 
-        const uint8_t EQMOD_TIMEOUT = 5;
-        const uint8_t EQMOD_MAX_RETRY = 3;
+        const long EQMOD_TIMEOUT = 200000; // us
+        const uint8_t EQMOD_MAX_RETRY = 75;
```

Comment:
Right, 75 is too many retries. Let me test on WiFI first to see how it goes.

Reply:
I will reduce this count, say to 10. Should I make the timeout a Property?

Reply:
Reduced to 10 for a total wait of 2 seconds.